src/hera.h

```diff
@@ -25,12 +25,24 @@
 #ifndef __HERA_H
 #define __HERA_H
 
+#if defined _MSC_VER || defined __MINGW32__
+# define HERA_EXPORT __declspec(dllexport)
+# define HERA_IMPORT __declspec(dllimport)
+#elif __GNU__ >= 4
+# define HERA_EXPORT __attribute__((visibility("default")))
```

Comment:
I think the default visibility is still "default", not "hidden".

Reply:
That's correct, I just added it to make it more explicit and clearer.